src/formatDate.js

```diff
@@ -8,7 +8,44 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splitSeparator = fromFormat[fromFormat.length - 1];
+  const splitDate = date.split(splitSeparator);
```

Comment:
The code assumes that the last element of `fromFormat` is the separator, which may not always be the case. Consider explicitly specifying the separator or handling cases where the separator is not the last element.